docs/src/man/basics.md

```diff
@@ -0,0 +1,2170 @@
+   3 │    14     28  female      2  rent     little           little
+                                                               3 columns omitted
+
+julia> german[:, [:Age, :Sex]]
+1000×2 DataFrame
+  Row │ Age    Sex
+      │ Int64  String
+──────┼───────────────
+    1 │    67  male
+    2 │    22  female
+    3 │    49  male
+    4 │    45  male
+    5 │    53  male
+    6 │    35  male
+    7 │    53  male
+    8 │    35  male
+  ⋮   │   ⋮      ⋮
+  994 │    30  male
+  995 │    50  male
+  996 │    31  female
+  997 │    40  male
+  998 │    38  male
+  999 │    23  male
+ 1000 │    27  male
+      985 rows omitted
+```
+
+In the example below we created a `DataFrame` having `:Sex` and `:Age` columns
+and the first five rows of the `german` data set:
+
+```jldoctest dataframe
+julia> german[1:5, [:Sex, :Age]]
+5×2 DataFrame
+ Row │ Sex     Age
+     │ String  Int64
+─────┼───────────────
+   1 │ male       67
+   2 │ female     22
+   3 │ male       49
+   4 │ male       45
+   5 │ male       53
+```
+
+Pay attention that `german[!, [:Sex]]` and `german[:, [:Sex]]` return a data frame object,
+while `german[!, :Sex]` and `german[:, :Sex]` return a vector. In the first case, `[:Sex]` 
+is a vector, indicating that the resulting object should be a data frame. On the other hand,
+`:Sex` is a single symbol, indicating that a single column vector should be extracted. Note 
+that in the first case a vector is required to be passed (not just any iterable), so e.g. 
+`german[:, (:Age, :Sex)]` is not allowed, but `german[:, [:Age, :Sex]]` is valid.
+
+```jldoctest dataframe
+julia> german[!, [:Sex]]
+1000×1 DataFrame
+  Row │ Sex
+      │ String
+──────┼────────
+    1 │ male
+    2 │ female
+    3 │ male
+    4 │ male
+    5 │ male
+    6 │ male
+    7 │ male
+    8 │ male
+  ⋮   │   ⋮
+  994 │ male
+  995 │ male
+  996 │ female
+  997 │ male
+  998 │ male
+  999 │ male
+ 1000 │ male
+985 rows omitted
+
+julia> german[!, [:Sex]] == german[:, [:Sex]]
+true
+
+julia> german[!, :Sex]
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+
+julia> german[!, :Sex] == german[:, :Sex]
+true
+```
+
+As it was explained above the difference between using `!` and `:`
+when passing a row index is that `!` does not perform a copy of columns, while `:` does.
+Therefore `german[!, [:Sex]]` data frame stores the same vector as the source `german` data frame,
+while `german[:, [:Sex]]` stores its copy. The `!` selector normally should be avoided
+as using it can lead to hard to catch bugs. However, when working with very large data frames
+it can be useful to save memory and improve performance of operations.
+
+
+Recapping what we have already learned,
+given the data frame `german` here are various ways to grab one of its columns as vector:
```

Comment:
better remove printing of the output but instead list the options, e.g. bulleted explaining what each of them does (now you do not list all options).

Reply:
Sorry, I did not get this completely. Can you provide me working example, please?

Reply:
I suggested not a working example but a list. The list would be:

To get the column `:Age` from the `german` data frame you can do the following:
* to copy the vector: `german[:, :Age]`, `german[:, "Age"]` or `german[:, 2]`;
* to get a vector without copying: `german.Age`, `german."Age"`, `german[:, :Age]`, `german[:, "Age"]` or `german[:, 2]`.